node/cn/backend.go

```diff
@@ -320,8 +321,18 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 		}
 	}
 
+	var resetFn func()
```

Comment:
This could lead to nil pointer dereference. You need to set empty function to this variable to avoid that.

Reply:
@kjhman21 This `restartFn` will be used if it is not nil as below code.
So It is ok. 

```
if self.restartFn != nil && self.restartTimeOut > 0 {
```